contracts/liquidator/vendor/@uniswap/v3-periphery/contracts/base/PeripheryPayments.sol

```diff
@@ -11,10 +11,6 @@ import '../libraries/TransferHelper.sol';
 import './PeripheryImmutableState.sol';
 
 abstract contract PeripheryPayments is IPeripheryPayments, PeripheryImmutableState {
-    receive() external payable {
-        require(msg.sender == WETH9, 'Not WETH9');
-    }
```

Comment:
Is some other contract sending ETH now?

Reply:
yep, Curve has pools that swap directly to ETH, so you receive ETH as part of the swap